amf-api-contract/shared/src/main/scala/amf/apicontract/internal/transformation/stages/WebApiReferenceResolutionStage.scala

```diff
@@ -45,6 +45,13 @@ class WebApiReferenceResolutionStage(keepEditingInfo: Boolean = false)
               copy.withId(sourceServer.id).withName(sourceServer.name.value())
             case _ => domain
           }
+        case sourceServerVariable: Parameter =>
```

Comment:
Add the check for the annotation DeclaredServerVariable

Reply:
done